Use mermaid & jquery from npm instead of own copies

piranna opened the pull request.

piranna committed:
Use mermaid & jquery from npm instead of own copies

piranna committed:
Clean-up of converter

JozoVilcek commented:
This looks nice. I would prefer to keep code, which converts mermaid stuff in separate process (phantom) in that separate `convert` folder. Could you please keed it there + remove npm debug log from merge? I will be more than happy to merge it then.

piranna commented:
Sorry, I missed this one. It's standard to have `npm-debug.log` in `.gitignore` file to prevent it for being uploaded. Also, I've removed the `convert` folder since now that all assets are removed the only item on the `convert` folder is `convert.html`, that's somewhat redundant.

piranna commented:
Any update on this?

JozoVilcek commented:
Uff, I am sorry for being so late. I missed this. Git now complains about merge conflicts. Can you please update this.

Commit:
fix: package.json to reduce vulnerabilities

Commit:
fix: package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:jquery:20160529

Latest report for piranna/gitbook-plugin-mermaid:
https://snyk.io/test/github/piranna/gitbook-plugin-mermaid

Commit:
fix: package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:concat-stream:20160901

Latest report for piranna/gitbook-plugin-mermaid:
https://snyk.io/test/github/piranna/gitbook-plugin-mermaid

piranna committed:
Merge pull request #3 from piranna/snyk-fix-a75eda7a

[Snyk Update] New fixes for 1 vulnerable dependency path

piranna committed:
Merge pull request #2 from piranna/snyk-fix-a4e943fa

[Snyk Alert] Fix for 1 vulnerable dependency path

piranna committed:
Merge pull request #1 from piranna/snyk-fix-dcfa46b0

[Snyk Alert] Fix for 1 vulnerable dependency path
